### ISSUE TITLE
Use other libs

### ISSUE THREAD

**Issue by ductm104** [ebadrian/metadl#8]
Hi, I see that your `requirements.txt` file only contains tensorflow based libraries.

How can I use other libs like `pytorch`?

Can you provide an example to use `torch` instead of `tf`. I just checked `maml_torch` but it doesn't work.

**Comment by ebadrian** [comment]
Hello! 
The requirements has been updated and now has the higher (second order optimization library required by maml_torch). The maml_torch baseline should now work with the conda environment. 
Everything you need to know to use torch models instead of TF is inside this baseline. More specifically, you only need to implement the torch data wrapper in order to use any torch models you want.  
Let me know how it goes!

**Comment by ductm104** [comment]
Hi @ebadrian ,

I have another question about how to get exactly the total number of meta class in meta training phase.

**Comment by ebadrian** [comment]
Hi! 
We plan to release this feature soon, I'll keep you updated.

**Comment by ductm104** [comment]
Hi @ebadrian , are there any updates on this problem? Because I cannot use 'batch' mode with CE Loss if `total_meta_class` is not known.
